<?xml version="1.0"?>

<launch>
  <include file="$(find final_assignment)/launch/simulation_gmapping.launch"/>
  <include file="$(find final_assignment)/launch/move_base.launch"/>

  <node name="driving_mode" pkg="final_assignment" type="driving_mode.py" required="true" output="screen" launch-prefix="xterm -e"/>

  <node name="driver_assistance" pkg="final_assignment" type="driver_assistance_node" respawn="true">
    <remap from="/base_scan" to="/scan"/>
  </node>

</launch>
